Debian typically builds natively, so disable forced use of
cross-compile.

Index: u-boot/arch/powerpc/config.mk
===================================================================
--- u-boot.orig/arch/powerpc/config.mk
+++ u-boot/arch/powerpc/config.mk
@@ -3,10 +3,6 @@
 # (C) Copyright 2000-2010
 # Wolfgang Denk, DENX Software Engineering, wd@denx.de.
 
-ifeq ($(CROSS_COMPILE),)
-CROSS_COMPILE := ppc_8xx-
-endif
-
 CONFIG_STANDALONE_LOAD_ADDR ?= 0x40000
 LDFLAGS_FINAL += --gc-sections
 LDFLAGS_FINAL += --bss-plt
